draft-ounsworth-pq-composite-sigs.mkd

```diff
@@ -93,11 +94,10 @@ The deployment of composite public keys and composite signatures using post-quan
 - Algorithm strength uncertainty: During the transition period, some post-quantum signature and encryption algorithms will not be fully trusted, while also the trust in legacy public key algorithms will start to erode.  A relying party may learn some time after deployment that a public key algorithm has become untrustworthy, but in the interim, they may not know which algorithm an adversary has compromised.
 - Backwards compatibility: During the transition period, post-quantum algorithms will not be supported by all clients.
 
-This document provides a mechanism to address algorithm strength uncertainty by providing formats for encoding multiple public keys and multiple signature values into existing public key and signature fields, as well as an algorithm for validating a composite signature. The issue of backwards compatibility is left open to be addressed in separate draft(s).
+This document provides a mechanism to address algorithm strength uncertainty by providing formats for encoding multiple public keys and multiple signature values into existing public key and signature fields, as well as an algorithm for validatinfg a composite signature. The issue of backwards compatibility is left open to be addressed in separate draft(s).
 
-This document is intended for general applicability anywhere that public key structures or digital signatures are used within PKIX structures.
+This document is intended for general applicability anywhere that public key structures or digital signatures are used within PKIX structures. While the CompositePublicKey structure defined herein is equally applicable to asymmetric encryption keys, this document is intentionally restricted to signatures. The combiner functions for composite encryption and KEMs is being addressed in other drafts such as [I-D.draft-stebila-tls-hybrid-design].
```

Comment:
I don't know if the reference to the steblia draft is appropriate.  It doesn't deal with composite public key encryption at all, and as for KEMs, it's solution is TLS-specific, and may be insecure in other scenarios.

Reply:
hmmm... Maybe we don't need this text at all? It's quite clear that this document is about signatures and if this can be reused somewhere else depends on a context.
I think Appendix B of Stebila draft (and this https://eprint.iacr.org/2018/903.pdf) talks a bit about some design considerations, but indeed in different context. But are there really some other drafts which address this problem?

Reply:
Removing the reference to Stebila makes sense.

I feel that it's still worth mentioning that while you could use this draft to create composite encryption certs, the semantics of that are left undefined by this draft.

Reply:
... maybe change the sentence to ".. are beyond the scope of this document" and leave it vague.